treeherder/log_parser/parsers.py

```diff
@@ -392,6 +392,7 @@ class ErrorParser(ParserBase):
         r"| error R?C\d*:"
         r"|ERROR [45]\d\d:"
         r"|mozmake\.exe(?:\[\d+\])?: \*\*\*"
+        r"|mozmake\.EXE(?:\[\d+\])?: \*\*\*"
```

Comment:
Perhaps combine this with the previous line using eg `(?:exe|EXE)` ?
Also it would be good to add a test to:
https://github.com/mozilla/treeherder/blob/4238fca43655caef566f0af3d1ff6d76a6dea41f/tests/log_parser/test_error_parser.py#L5